app/build.gradle

```diff
@@ -207,6 +207,9 @@ dependencies {
     implementation "com.jakewharton.rxbinding2:rxbinding:2.2.0"
 
     implementation "org.ocpsoft.prettytime:prettytime:4.0.5.Final"
+
+    implementation 'commons-io:commons-io:2.7'
```

Comment:
These versions require java 8 which might not work on older devices.
Try to use version `2.5` of commons-io, `3.5` of commons-lang for compatibility.

Reply:
That's not an issue anymore with Android Studio 4